template/_stylelint.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  rules: {
+
```

Comment:
If there is no default config is needed here, can we add some descriptions and doc link here ?

Reply:
Yes I agree.
Is there any expected stylelint configuration for nuxt like `@nuxtjs/styleint-config`?

Reply:
I think default config is OK, maybe just adding desc and links for detailed config.

Reply:
@clarkdo I added a little description
```js
// add your custom config here
// https://stylelint.io/user-guide/configuration
```